src/components/OfflineWithFeedback.js

```diff
@@ -31,7 +31,7 @@ const propTypes = {
     errors: PropTypes.object,
 
     /** A function to run when the X button next to the error is clicked */
-    onClose: PropTypes.func.isRequired,
```

Comment:
In what case would we want to not pass this function?

Reply:
Here I wrapped the text inputs in OfflineWithFeedback so that they would be grayed out, and I used a separate OfflineWithFeedback around the Save button to show the error. @tgolen recommended that the prop should no long be required and default to a no op.
https://github.com/Expensify/App/blob/021f1ff65d901fc8f97e75f3cdca1f776372403e/src/pages/workspace/WorkspaceSettingsPage.js#L149-L151

Reply:
Don't quite get it... so how/when will be the error in that field cleared?

Reply:
The OfflineWithFeedback component that displays the error passes an onClose prop.
https://github.com/Expensify/App/blob/b2d2cd3515b9bb4e8da48df75dd3fdc852002062/src/pages/workspace/WorkspaceSettingsPage.js#L107-L110

Reply:
But isn't the one without the onClose also showing an X button that does nothing when clicked? :confused:

Reply:
The one without `onClose` doesn't pass `errors` and therefore it will not render the error:
https://github.com/Expensify/App/blob/b2d2cd3515b9bb4e8da48df75dd3fdc852002062/src/components/OfflineWithFeedback.js#L103

Reply:
Ahhhhh! ok ok got it, thanks for the patience in explaining it to me